### PR TITLE
Whitelist PhotoPrism WebDAV endpoints

### LINKED ISSUE #57: PhotoPrism WebDAV access
Currently the App Proxy is protecting the WebDAV endpoints within PhotoPrism.

It appears safe to whitelist: `/originals` and `/import` as these are protected by HTTP Basic Auth by Photoprism.

More details here: https://docs.photoprism.app/user-guide/sync/webdav/#server-url

### PR DESCRIPTION
**Pull request by nevets963**
Closes #57

### CODE REVIEW

File: photoprism/docker-compose.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ services:
     environment:
       APP_HOST: $APP_PHOTOPRISM_IP
       APP_PORT: $APP_PHOTOPRISM_PORT
+      PROXY_AUTH_WHITELIST: "/originals/*,/import/*"
```
Comment (tsnieman) [inline photoprism/docker-compose.yml:8]: These endpoints have been [confirmed by the developers of PhotoSync](https://mobile.twitter.com/photosync_app/status/1542509638477680642) (which is the [recommended app for uploading to PhotoPrism](https://docs.photoprism.app/user-guide/sync/mobile-devices/)).

Comment (tsnieman) [reply photoprism/docker-compose.yml:8]: WYT about documenting _why_ these are whitelisted within this file?

```suggestion
      # These endpoints are protected by HTTP Basic Auth by Photoprism.
      # More details here: https://docs.photoprism.app/user-guide/sync/webdav/#server-url
      PROXY_AUTH_WHITELIST: "/originals/*,/import/*"
```

Comment (lukechilds) [reply photoprism/docker-compose.yml:10]: Great suggestion, thanks @tsnieman!

**Review by lukechilds — APPROVED** [review]

File: photoprism/docker-compose.yml [hunk 2]
```diff
@@ -5,6 +5,9 @@ services:
     environment:
       APP_HOST: $APP_PHOTOPRISM_IP
       APP_PORT: $APP_PHOTOPRISM_PORT
+      # These endpoints are protected by HTTP Basic Auth by Photoprism.
+      # More details here: https://docs.photoprism.app/user-guide/sync/webdav/#server-url
+      PROXY_AUTH_WHITELIST: "/originals/*,/import/*"
   web:
     image: photoprism/photoprism:20210925@sha256:238e6955804d82097d08f0e4318835721f46bd844f9065a38aa7d112ab72906e
```
Comment (lastzero) [inline photoprism/docker-compose.yml:12]: This version is almost a year old? You might want to update it from time to time 🤗

👉 https://docs.photoprism.app/release-notes/

**Pull request merged**